easybuild/easyconfigs/j/JasPer/JasPer-2.0.12-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,28 @@
+# Contributed by Jean-Guillaume Piccinali and Samuel Omlin (CSCS)
```

Comment:
jg@cscs please

Reply:
or just jg, thanks

Reply:
Done. I had copied it from another .eb file...